Workspaces/ProjectWorkspace/Scripts/FolderListViewController.cs

```diff
@@ -18,10 +18,10 @@ sealed class FolderListViewController : NestedListViewController<FolderData, Fol
 		[SerializeField]
 		Material m_ExpandArrowMaterial;
 
-		string m_SelectedFolder;
-
 		readonly Dictionary<string, bool> m_ExpandStates = new Dictionary<string, bool>();
 
+		public string selectedFolder { get; private set; }
```

Comment:
Should we replace SelectFolder with a setter on selectedFolder? I don't like having both at the same time. If we want to keep setFolder as a method, we should instead use a getFolder method/delegate.